site/themes/carvel/assets/scss/_components.scss

```diff
@@ -43,6 +43,7 @@
             .grid.three {
                 padding-bottom: 60px;
                 .card {
+                    border: 1px solid $lightgrey;
```

Comment:
This is going to add a border to all usage of `.grid.three.card` which is also being used on the home page. It's subtle but it does have a slight side effect of stretching out the cards which _I think_ is undesirable.

I would consider using a CSS id to distinguish the resources page `.grid.three.card` usage from the home page usage. I'd be happy to chat about this further if you're not familiar with CSS.

Reply:
I guess you could still use a CSS class for this. Just need some way to differentiate the usage between the resources page from other pages.